### DISCUSSION TITLE
Forward Port 8123 (Dynmap) Through Container

### DISCUSSION THREAD

**Discussion by ethanic17** [itzg/docker-minecraft-server#2226] [General]
I installed Dynmap but am a bit confused on how to open up the port so that the webpage actually works. Would I need to create a new container just to host Dynmap or can I just forward it through the` docker-compose.yml` file (see screenshot below & commented out part 8123). And if that actually works, re-creating the container via docker-compose won't affect the server files as I've attached a data volume correct?

![image](https://github.com/itzg/docker-minecraft-server/assets/41976386/3e1a2b8f-0806-49a9-87aa-5d468140d539)

**Comment by itzg** [comment]
I haven't used dynmap myself, but from what I see in the docs you should just be able to uncomment the 8123 port entry. Was there a reason you commented it out?

After making changes to the compose file re-run `docker compose up -d`. Correct, your data will be preserved. 

BTW, no need to use a custom server, just use the built in Paper support: https://docker-minecraft-server.readthedocs.io/types-and-platforms/server-types/paper/

**Reply by ethanic17** [reply]
Yup, it worked! Just wanted to double check just in case something bad happened lol (I was thinking somehow minecraft might route to 8123 instead or something catastrophic).  And I'm using an older version of Paper though (1.16.3), is there an environment variable to specify which version of Paper to use?

**Reply by itzg** [reply]
> is there an environment variable to specify which version of Paper to use? 

Yes, `VERSION` sets the Minecraft version for all server types.

**Reply by ethanic17** [reply]
ah sweet, will change that right now, thanks!
